.gitignore

```diff
@@ -0,0 +1,37 @@
+# OSX
+#
+.DS_Store
```

Comment:
The file was automatically generated by react-native, but (again) you're
right !